angular/src/app/app.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, ViewChild } from '@angular/core';
+import { DxSchedulerComponent } from 'devextreme-angular';
+import DataSource from 'devextreme/data/data_source';
+import { Service } from './app.service';
+
+import { ClickEvent } from 'devextreme/ui/button';
+import { dxSchedulerAppointment } from 'devextreme/ui/scheduler';
+
+
+
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+  providers: [Service],
+})
+export class AppComponent {
+  @ViewChild('targetScheduler', { static: true })
+  scheduler: DxSchedulerComponent;
+  dataSource = new DataSource({
+      store: this.service.getData(),
+   });
```

Comment:
Let's add empty line here, as we do it in another places:

```suggestion
  scheduler: DxSchedulerComponent;
  
  dataSource = new DataSource({
      store: this.service.getData(),
   });
```

Reply:
resolved